core/src/main/java/org/mskcc/cbio/oncokb/util/AnnotationSearchUtils.java

```diff
@@ -112,6 +112,13 @@ private static List<TypeaheadSearchResp> findMatchingCancerTypes(String query, B
         if (exactMatch) {
             matchedTumorTypes.add(ApplicationContextSingleton.getTumorTypeBo().getByName(query));
         } else {
+            for (Map.Entry<String, TumorType> currSubtype : CacheUtils.getCodedTumorTypeMap().entrySet()) {
+                String code = query.toUpperCase();
+                if (currSubtype.getKey().equals(code)) {
```

Comment:
The code should never have lowercase, so don't need to convert the key to uppercase here.

Reply:
@calvinlu3 this is query though. I imagine if user types in `nsclc`, I want it to be matched to NSCLC.

Reply:
Sorry, should clarify that I meant the code from `currSubtype.getKey()`. I assume any future cancer types that we may add to database should not have a lowercase in the code right?

Reply:
Presumably